### DISCUSSION TITLE
Conda stuck solving environment

### DISCUSSION THREAD

**Discussion by rtaylo45** [idaholab/moose#19196] [Q&A Getting Started] [answered]
Hello,

Im having trouble installing moose-tools and moose-libmesh on a linux cluster for continuous integration. I have followed the steps on the installation guide and when i get to the step:

```
conda install moose-tools
conda install moose-libmesh
```

where It gets stuck on `Solving environment: ...working...`  for hours. I have tried googling this [error](https://stackoverflow.com/questions/63734508/stuck-at-solving-environment-on-anaconda) but i have found no solution which works. I have also tried the previous solution found on the [moose form](https://github.com/idaholab/moose/discussions/17094). I have tried installing libmesh first but it has not worked, and using conda clean. The CI has a time limit of 6 hours before it kills the process. Anyone have ideas for how to fix this?

Thanks,
Zack

**Comment by loganharbour** [comment, marked as answer]
Try this instead:

```
conda update --all
conda install mamba
mamba install moose-tools moose-libmesh
```

**Reply by rtaylo45** [reply]
I get an error when installing mamba:

```
Preparing transaction: done
Verifying transaction: done
Executing transaction: failed
ERROR conda.core.link:_execute(502): An error occurred while installing package 'conda-forge::colorama-0.4.4-pyh9f0ad1d_0'.
OSError(2, 'No such file or directory')
Attempting to roll back.
```

**Reply by GiudGiud** [reply]
Ah weird. Can you try again later? There s flukes with conda.
If it doesnt work can you try again from a fresh environment?

**Reply by rtaylo45** [reply]
My boss was able to get it to work. Thanks for the help, installing with mamba did the trick
